tests/jenkins/BuildPipelineTest.groovy

```diff
@@ -36,11 +37,11 @@ abstract class BuildPipelineTest extends CommonPipelineTest {
 
         helper.registerSharedLibrary(
             library().name('jenkins')
-                .defaultVersion('<notNeeded>')
+                .defaultVersion('1.0.0')
```

Comment:
default needs to be `main` and each jenkinsfile needs to have a override with what version the file is using, What do you think @prudhvigodithi ?

Reply:
makes sense! Sayali, I will make the defaultVersion to 'main'  on [BuildPipelineTest.groovy], and override to version 1.0.0 on each individual jenkinsfile. thanks,